instance/migrations/0069_openstack_settings.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-01-17 06:55
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.contrib.postgres.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0068_security_groups'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='openedxinstance',
+            name='openstack_server_base_image',
+            field=django.contrib.postgres.fields.JSONField(blank=True, default=None, help_text='JSON openstack base image selector, e.g. {"name": "ubuntu-12.04-ref-ul"} Defaults to settings.OPENSTACK_SANDBOX_BASE_IMAGE on server creation.', null=True),
```

Comment:
Thanks for reminding me, @haikuginger, and for the documentation pointers, @bradenmacdonald.  I've added the data migration with 32a1a76.